Makefile.toml

```diff
@@ -225,6 +225,17 @@ fi
 tail -f -n 5 ${PREFIX_LOG}/$1
 '''
 
+[tasks.ci-clean-logs]
+private = true
+category = "RiseDev - CI"
+description = "Clean logs before CI"
+script = '''
+#!/usr/bin/env bash
+set -e
+cat /risingwave/.risingwave/log/compactor-6660.log | sed -E 's/table_ids: \[[0-9, ]+\]/table_ids: [hidden]/g' > tmp.log
```

Comment:
Is it possible to disable logs for compactor in advance through `RUST_LOG`?

Reply:
Well I sort of prefer this approach, as it pinpoints and mitigates the exact cause.
If we disable the compactor logs entirely, means we can't check it if there's some error due to compactor.

Reply:
No need to disable everything, just the verbose lines. 😄

Reply:
Fixed it 👍